force-app/main/default/lwc/courseRegistrationInformation/courseRegistrationInformation.html

```diff
@@ -0,0 +1,29 @@
+<template>
+    <lightning-record-view-form record-id={courseId} object-api-name="Course__c">
```

Comment:
```suggestion
    <lightning-record-view-form if:true={courseId} record-id={courseId} object-api-name="Course__c">
```

Da skjules ALT hvis record ID ikke finnes.